xapian-letor/ranker/listnet_ranker.cc

```diff
@@ -193,6 +193,6 @@ ListNETRanker::rank_fvv(const std::vector<FeatureVector> & fvv) const {
 	    listnet_score += fvals[j] * parameters[j];
 	testfvv[i].set_score(listnet_score);
     }
-    std::sort(testfvv.begin(), testfvv.end(), &Ranker::scorecomparer);
+    //std::sort(testfvv.begin(), testfvv.end(), &Ranker::scorecomparer);
```

Comment:
Just delete this line; you don't need to keep commented code, because we can find it from the git history if we need.